### DISCUSSION TITLE
iea 15MW Turbine Rated Speed (V3)

### DISCUSSION THREAD

**Discussion by JackDI1** [NatLabRockies/floris#952] [Q&A] [answered]
Hi all,

Is there a quick and easy way to identify the turbine rated speed for the iea_15MW turbine in v3? I have had a look through the repo and it doesn't look like it is explicitly defined anywhere, but I may be missing something obvious, please let me know if that is the case.

Thanks,
Jack

**Comment by Bartdoekemeijer** [comment, marked as answer]
I would say the easiest way is to just check the corresponding documentation of the IEA 15MW turbine. The rated wind speed for the IEA 15MW turbine is defined to be at 10.59 m/s. See the report here: https://www.nrel.gov/docs/fy20osti/75698.pdf. Hope this helps!

**Reply by JackDI1** [reply]
That is perfect thank you!
